elastic/kibana
# Console Autocomplete Overrides for 7.11

Closes #86586 

## Summary

This PR is to update the autocomplete suggestions for new API endpoints and parameters introduced in 7.11.

## How to test

1. Navigate to the Dev Console.
2. Enter the different endpoints that were updated in the PR. 
3. After adding the endpoint for the request, check to see if the different parameters (whether URL or Body) show up in their respective places.
